cv_next/package.json

```diff
@@ -42,12 +44,15 @@
     "react-intersection-observer": "^9.5.3",
     "react-loading": "^2.0.3",
     "server-only": "^0.0.1",
-    "sharp": "^0.33.3",
+    "sharp": "^0.33.4",
+    "swr": "^2.2.5",
     "tailwind-merge": "^1.14.0",
     "tailwindcss": "3.3.3",
     "tailwindcss-animate": "^1.0.6",
     "types": "^0.1.1",
-    "utf-8-validate": "^6.0.3"
+    "utf-8-validate": "^6.0.3",
+    "winston": "^3.13.0",
```

Comment:
Make sure we're ending up with the correct logging library when finishing merging all the PRs.